pages/glossary/main-block-of-content.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Main block of content
+key: main-block-of-content
+unambiguous: true
+objective: false
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+The _main block of content_ of an [HTML web page][] is a [block of content][] whose content fulfills the expectations of users navigating to the page.
```

Comment:
Yes, the definition is subjective (in this case literally depending on the subject, i.e. the user navigating to the page).
This shouldn't be a problem since it's only used in Expectations.

Reply:
So how do I determine a user's expectation? Do I do it based on the page title? Based on link text of pages that link to this page, and if so, how do I come up with a list of pages that link to this page? If there are multiple, do I allow multiple possible expectations, thus allowing different things to be "main content"? Do I based this on visuals cues the page gives me? Do I use landmarks, exclusively, or maybe in combination with other cues?

There are lots of ways to do this, and this definition doesn't tell us which one to use. That makes this ambiguous.

Reply:
I'm confused and I feel we're not talking about the same thing 😕 

"Main content" (informal meaning) is about content, not about visual or logical (landmark, heading elements, …) cues.

Or rather, the cues are used to tell user where the main content is, but they do not define it. In a sense, visual cues (as in putting the main content in a large "column" in the middle of the page) is a way to satisfy 2.4.1 for sighted users (they know they can skip the small column on the side and focus on the big stuff in the middle). But it is not the main content because it is in the middle, it is in the middle because it is the main content.
If the main content is not in the middle, or is hidden being an endless stream of "subscribe!" popups, or similar annoyance, I usually just stop reading the page and go away.

---
Looking at [HTML page title is descriptive](https://act-rules.github.io/rules/c4a8a4#expectation) we simply say that the `title` must describe "the topic or purpose of the overall content of the document." I don't think the title must describe the navigation menu or the search box, so I guess this is also a way to refer to the "main content".

Would you prefer something in the line of 
> The main block of content is a block whose topic or purpose is the same as the one of the document it is in

?

---
Also, rules using the def have an assumption that there is only one main block of content. I feel it is OK because I feel WCAG is making the same assumption (Understanding 2.4.1 talks about "direct access to **the** primary content of the Web page", emphasise mine).

---
@ShadowBB @carlosapaduarte you both approved this, do you have an opinion on this?
I'll tell @nitedog to put that on the agenda for Monday.

Reply:
Yeah, lets discuss Monday. This definition does not tell how to determine the user's expectation, and because there is more than one way to do it, that makes this ambiguous.

I don't really have a solution. This is definitely a difficult one to determine. My initial thoughts on this would be not to try, and to make this rule work exclusively by determining if blocks of repeating content can be skipped, because that is something we can determine unambiguously.